install.sh

```diff
@@ -110,6 +120,21 @@ prepare_user() {
   return 0
 }
 
+run_as_user() {
+  sudo -u $USER /bin/bash -c "
```

Comment:
We might be able to put additional file under `/etc/sudoers.d` directory to reset `secure_path`. Even so, I feel the string way is slightly better.